testing/set-env-vars.sh

```diff
@@ -23,6 +29,8 @@ DISABLE_WEBHOOK_FILTER_FOR_TESTS=${DISABLE_WEBHOOK_FILTER_FOR_TESTS:="false"}
 
 export KUBEBUILDER_ASSETS
 export USE_EXISTING_CLUSTER
+export SINGLE_TEST_TIMEOUT
+export SUITE_TIMEOUT
```

Comment:
nit: I think `export` can be done at the time of variable initialization

Reply:
yes, one-liners will make the file shorter.

Reply:
So should I refactor the whole file to make it consistent?

Reply:
yes